internal/shim-sgx/layout.ld

```diff
@@ -0,0 +1,77 @@
+ENTRY(_start)
+
+PHDRS {
+    rodata PT_LOAD FILEHDR PHDRS;
+    data PT_LOAD;
+    text PT_LOAD;
+    dynamic PT_DYNAMIC;
+    note PT_NOTE;
+
+    stk0 PT_LOAD;
+    tcs0 PT_LOAD FLAGS(1 << 20); /* PF_ENARX_SGX_TCS */
+    ssa0 PT_LOAD;
+
+    exec 0x634A0003 FLAGS(0); /* PT_ENARX_EXEC */
```

Comment:
`PT_ENARX_EXEC` or `PT_ENARX_CODE` or `PT_ENARX_INIT` ??

Reply:
I propose we rename the "code" layer to the executable or "exec" layer.